ui/widget.js

```diff
@@ -246,7 +257,7 @@ $.widget.bridge = function( name, object ) {
 	};
 };
 
-$.Widget = function( /* options, element */ ) {};
+$.Widget = function( /* Options, element */ ) {};
```

Comment:
@scottgonzalez It DID catch this which is why it became uppercase. 

@markelog @mikesherov It seems like the `requireCapitalizedComments` is completely unenforceable in a realistic sense all of these valid comments fail with it 

```
// http://google.com

// #3456 this does xxx
// to yyy so that zzz.

function( /* option */ ) {

/*

test(  "i dont want this to run", function() { 

*/
```

There are probably more cases then just this, but because of these ones I was forced to disable this rule within jQuery UI ( though i believe i got most if not all the valid comments fixed before disabling )